VARIABLES.md

```diff
@@ -1786,7 +1826,7 @@ Below are the supported variables for the role confluent.ksql
 
 ### ksql_custom_log4j
 
-Boolean to enable cp-ansible's Custom Log4j Configuration
+Boolean to reconfigure ksqlDB's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure ksqlDB's logging with the RollingFileAppender and log cleanup functionality.